File: Assets/Objects/House.cs
using UnityEngine;

public class House : MonoBehaviour
{
    public ObjectLoop ObjectLoopScript;
    public bool MomSpaceTaken = false;
    public bool ManSpaceTaken = false;
    public GameObject MomObject;
    public GameObject ManObject;

    // Use this for initialization
    void Start()
    {
        ObjectLoopScript = GameObject.Find("System").GetComponent<ObjectLoop>();
        ObjectLoopScript.Houses.Add(this.gameObject);
    }

    // Update is called once per frame
    void Update()
    {
    }
}

File: Assets/Objects/House.js
#pragma strict
var ObjectLoopScript : ObjectLoop;
var MomSpaceTaken : boolean = false;
var ManSpaceTaken : boolean = false;
var MomObject : GameObject;
var ManObject : GameObject;


function Start () 
{
	ObjectLoopScript = gameObject.Find("System").GetComponent(ObjectLoop);
	ObjectLoopScript.Houses.Add(this.gameObject);
}

function Update () {

}